integration/tests/main/09-user-positive-installation.spec.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 the Kubeapps contributors.
```

Comment:
FWIW, I wasn't thinking that we needed a new test for this, but rather just updating the existing default deployment test to use the `kubeapps-user`... we know if it passes with that that it will have no issues to pass with a higher-privileged account. Adding redundant tests will just lengthen the test run, won't it? Or is there a reason you think we should test both with the `kubeapps-user` as well as with `kubeapps-operator`?

Reply:
In the new `09` test, repository is created by admin user and consumed by another user with less privileges. I don't think we are covering that in other tests? I mean, in `03` test, admin does everything (add repo, deploy, etc.) and in test `04` admin deploys to `default` namespace using an initial repo from Kubeapps installation.
I think each test covers a slightly different scenario, it's a bit of the "how far do we want to get with e2e tests?" question I would say.
However, happy to remove any test if you think that is not needed!

Reply:
OK - I hadn't realised it was adding a repo. I'm OK with having a separate test, but it's not clear to me why we'd add a repo as part of this test (given we already have a repo with charts we can use for this test, and have other tests that add repositories). Also, I'm not sure about adding a non-bitnami repo so that our CI is now dependent on the prometheus community helm charts?

Reply:
> @absoludity Update: removing the rolebinding deletion from `.circleci/config.yml` works as expected.

Great, though it might be worth checking with @antgamdia - looks like he added the deletion. I can only guess that the intent was to use the `default` namespace for the kubeapps user and so extra RBAC for other namespaces was removed, at the time.

Reply:
That's right, though I can't 100% remember, the idea was just using the default namespace (perhaps because of the problems you mentioned about the test selecting the wrong context?), so I just deleted this extra RBAC (which was granting `edit` ClusterRole in the `kubeapps-user-namespace` namespace
Now the tests are handling better how to switch contexts, +1 to remove any deletion that might veil any errors on the current rbac dev file we're using.